include/sprite.h

```diff
@@ -76,8 +78,8 @@ struct UNK_0808B3FC_UNK240 {
 
     u16 unk8;
 
-    // assetId or animIndex, not sure
-    u16 unkA;
+    // anim
+    AnimId unkA;
```

Comment:
You can set this on UNK_080E0D64 too

Reply:
I'll update the "u16" arrays in the interactables later as well. But will be part of another PR. :)